PocketKit/Sources/Sync/Requests.swift

```diff
@@ -143,6 +143,47 @@ public enum Requests {
         return request
     }
 
+    public static func fetchCollectionAuthor(by name: String) -> NSFetchRequest<CollectionAuthor> {
+        let request = CollectionAuthor.fetchRequest()
+        request.predicate = NSPredicate(format: "name = %@", name)
+        request.fetchLimit = 1
+        return request
+    }
+
+    public static func fetchCollectionAuthors(by slug: String) -> NSFetchRequest<CollectionAuthor> {
+        let request = CollectionAuthor.fetchRequest()
+        request.predicate = NSPredicate(format: "collection.slug = %@", slug)
+        return request
+    }
+
+    public static func fetchCollectionStory(by url: String) -> NSFetchRequest<CollectionStory> {
+        let request = CollectionStory.fetchRequest()
+        request.predicate = NSPredicate(format: "url = %@", url)
+        request.fetchLimit = 1
+        return request
+    }
+
+    public static func fetchCollectionStories(by slug: String) -> NSFetchRequest<CollectionStory> {
+        let request = CollectionStory.fetchRequest()
+        request.predicate = NSPredicate(format: "collection.slug = %@", slug)
+        request.sortDescriptors = [NSSortDescriptor(keyPath: \CollectionStory.sortOrder, ascending: true)]
+        return request
+    }
```

Comment:
do we want to remove? curious to see why we would use this over the `RichFetchRequest`